src/index.tsx

```diff
@@ -1,19 +1,127 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+
+const Counter = (props) => {
```

Comment:
A) In such cases please use object distructuring on component level.

B) `src/components/...`
1) Please create a new folder `src/components` and inside create folder for each single component. 
2) Then import and use them here.